### DISCUSSION TITLE
关于 哔哩 后续更新的说明

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#857] [通知]
目前暂时开放商店版本的下载：
`ms-windows-store://pdp/?productid=9mvn4nslt150`

*复制上面的代码到浏览器地址栏打开，即可在 Windows 11 应用商店中下载*

---

在商店渠道开放期间，对于哔哩的更新计划将做出以下调整：

1. Github 版本 和 商店版本 不共享应用证书，这意味着两个版本可以共存
2. 两个渠道的版本共用同一套版本标识，即 `2.{年月}.{构建号}.0`，比如 `2.2203.3.0` 表示 2022年3月的第三次构建
3. Github 版本 将作为尝鲜版 （类似 Windows insider），不定期更新。商店版本作为稳定版，月末更新（如果当月有更新的话）
4. Github 版本 拥有自己的更新通知，商店版本不会接收到来自 Github 的更新提醒

---

这是我在项目管理方面做出的新的尝试，以实践自己在这方面学习到的东西。
还是那句话，练手，都是练手 🤷‍♂️。

**Comment by Atlas124** [comment]
66666😍
